### ISSUE TITLE
Mimikatz and WCE

### ISSUE THREAD

**Issue by tekwizz123** [pwnwiki/pwnwiki.github.io#102]
Just thought of this whilst going though "AT is the New Black", but do we actually have any decent documentation on Mimikatz and WCE? If not it would be good to add in some good documentation on these two tools to our post exploitation for Windows section as they tend to come in handy on a pentest.

**Comment by WebBreacher** [comment]
Yes. This would be great or at least point to other resources on the Interwebs that show how to use them.

**Comment by mubix** [comment]
Ya there is a serious lack of documentation on Mimikatz and WCE alike

**Comment by Chron1cle** [comment]
Check these out:

WCE:
     http://www.ampliasecurity.com/research/wcefaq.html#whatiswce
Mimikatz
     https://www.offensive-security.com/metasploit-unleashed/mimikatz/

**Comment by jakxx** [comment]
Working on compiling a list of links. Do we want this under the "presence" heading?
